src/zenml/integrations/kubeflow/flavors/kubeflow_orchestrator_flavor.py

```diff
@@ -116,6 +121,37 @@ def _migrate_pod_settings(cls, values: Dict[str, Any]) -> Dict[str, Any]:
             values["node_affinity"] = {}
             values["node_selectors"] = {}
 
```

Comment:
Maybe we can put this into a separate root validator so it's a little cleaner? If not, the method name should change at least

Reply:
I did not want to get into the area of which root validator is called first, so I have for now just renamed the method